public/locales/pl/common.json

```diff
@@ -43,7 +43,7 @@
   "custom": "Ustaw...",
   "timestamp": "Data i czas",
   "lastfmWillRejectOldTimestamps": "Last.fm odrzuci scrobble starsze niż dwa tygodnie.",
-  "keepOriginalTimestamp": "Scrobbluj używając oryginalnej daty i godziny.",
+  "keepOriginalTimestamp": "Scrobbluj zachowując teraźniejszą datę i godzinę.",
```

Comment:
That doesn't seem right. "Teraźniejsza" means **present** while this option, as I understand it, is specifically for creating a scrobble with a timestamp of a different (already existing) scrobble that's in **the past**.